doc/csv/recipes.rdoc

```diff
@@ -0,0 +1,330 @@
+== Recipes
+
+=== Contents
+- {Parsing}[#label-Parsing]
+  - {Parse from String Without Headers}[#label-Parse+from+String+Without+Headers]
+  - {Parse from String with Headers}[#label-Parse+from+String+with+Headers]
+  - {Parse from File Without Headers}[#label-Parse+from+File+Without+Headers]
+  - {Parse from File with Headers}[#label-Parse+from+File+with+Headers]
+  - {Parse from IO Stream Without Headers}[#label-Parse+from+IO+Stream+Without+Headers]
+  - {Parse from IO Stream with Headers}[#label-Parse+from+IO+Stream+with+Headers]
+- {Generating}[#label-Generating]
+  - {Generate to String Without Headers}[#label-Generate+to+String+Without+Headers]
+  - {Generate to String with Headers}[#label-Generate+to+String+with+Headers]
+  - {Generate to File Without Headers}[#label-Generate+to+File+Without+Headers]
+  - {Generate to File with Headers}[#label-Generate+to+File+with+Headers]
+  - {Generate to IO Stream Without Headers}[#label-Generate+to+IO+Stream+Without+Headers]
+  - {Generate to IO Stream with Headers}[#label-Generate+to+IO+Stream+with+Headers]
+- {Filtering}[#label-Filtering]
+  - {Filter String to String Without Headers}[#label-Filter+String+to+String+Without+Headers]
+  - {Filter String to String with Headers}[#label-Filter+String+to+String+with+Headers]
+  - {Filter String to IO Stream Without Headers}[#label-Filter+String+to+IO+Stream+Without+Headers]
+  - {Filter String to IO Stream with Headers}[#label-Filter+String+to+IO+Stream+with+Headers]
+  - {Filter IO Stream to String Without Headers}[#label-Filter+IO+Stream+to+String+Without+Headers]
+  - {Filter IO Stream to String with Headers}[#label-Filter+IO+Stream+to+String+with+Headers]
+  - {Filter IO Stream to IO Stream Without Headers}[#label-Filter+IO+Stream+to+IO+Stream+Without+Headers]
+  - {Filter IO Stream to IO Stream with Headers}[#label-Filter+IO+Stream+to+IO+Stream+with+Headers]
+
+=== Parsing
+
+==== Parse from \String Without Headers
+
+\Class method CSV::parse can read a source \String all at once,
```

Comment:
Could you use `#{CLASS_NAME}.#{CLASS_METHOD_NAME}` format?

```suggestion
\Class method CSV.parse can read a source \String all at once,
```

Reply:
Fixed (throughout).